sample_app/build.gradle

```diff
@@ -57,6 +61,8 @@ dependencies {
     implementation "androidx.core:core-ktx:$androidxCoreVersion"
     implementation "io.reactivex.rxjava3:rxjava:$rxJavaVersion"
     implementation project(':overlapping_panels')
+    implementation 'androidx.coordinatorlayout:coordinatorlayout:1.1.0'
```

Comment:
nit: Can we create a variable for `1.1.0` and `1.4.0` similar to what we do with `rxJavaVersion` and `androidxCoreVersion`?